tests/integration/SdkTest_test.cpp

```diff
@@ -5107,7 +5125,7 @@ TEST_F(SdkTest, SyncResumptionAfterFetchNodes)
     removeSync(sync4Path);
 
     // wait for the sync removals to actually take place
-    std::this_thread::sleep_for(std::chrono::seconds{20});
+    std::this_thread::sleep_for(std::chrono::seconds{5});
```

Comment:
Note the time reduction before merging. Is it intentionally commited?

Reply:
yes - the tests are pretty slow already, and 5 seconds should be more than enough :-)  thanks